code/datums/components/riding/riding_mob.dm

```diff
@@ -194,11 +194,11 @@
 		return
 
 	if(ride_check_flags & RIDER_NEEDS_ARMS) // piggyback
-		living_parent.log_message("started giving [rider] a piggyback ride", LOG_ATTACK, color="pink")
-		rider.log_message("started piggyback riding [living_parent]", LOG_ATTACK, color="pink")
+		living_parent.log_message("started giving [rider] a piggyback ride", LOG_GAME, color="pink")
+		rider.log_message("started piggyback riding [living_parent]", LOG_GAME, color="pink")
 	else if(ride_check_flags & CARRIER_NEEDS_ARM) // fireman
-		living_parent.log_message("started fireman carrying [rider]", LOG_ATTACK, color="pink")
-		rider.log_message("was fireman carried by [living_parent]", LOG_ATTACK, color="pink")
+		living_parent.log_message("started fireman carrying [rider]", LOG_GAME, color="pink")
+		rider.log_message("was fireman carried by [living_parent]", LOG_VICTIM, color="pink")
```

Comment:
Why is this `LOG_VICTIM` while the rest of the riding messages are `LOG_GAME`?

Reply:
to be in line with other instances, since the player doesn't have to be alive/awake to fireman carry but piggyback does require input from the rider

Reply:
shall I change it to game?

Reply:
nevermind i did already